lib-php/PlaceLookup.php

```diff
@@ -227,7 +227,7 @@ public function lookup($aResults, $iMinRank = 0, $iMaxRank = 30)
             }
             $sSQL .= '    avg(ST_X(centroid)) AS lon, ';
             $sSQL .= '    avg(ST_Y(centroid)) AS lat, ';
-            $sSQL .= '    COALESCE(importance,0.75-(rank_search::float/40)) AS importance, ';
+            $sSQL .= '    importance, ';
```

Comment:
When removing the on-the-fly default importance computation here, the mentioned issue arises